values/strings.xml

```diff
@@ -2276,14 +2284,16 @@ By connecting the game to a TheoTown Forum account you can participate in commen
     <string name="dialog_uploadregionconfirm_title">Upload now?</string>
     <string name="dialog_uploadregionconfirm_text">Ensure that you have a stable internet connection before starting the process.</string>
     <string name="dialog_uploadregionunsuited_title">Upload region</string>
-    <string name="dialog_uploadregionunsuited_text">As of right now you can only upload regions that are not part of the game nor have any cities in them (yet).</string>
+    <string name="dialog_uploadregionunsuited_text">As of right now you can only upload regions that have cities in them (yet).</string>
     <string name="dialog_acquirecityaccount_title">Account needed</string>
     <string name="dialog_acquirecityaccount_text">In order to acquire a plot in a online region you have to connect TheoTown with a TheoTown account.</string>
     <string name="region_onlinecity_free">This plot is free so you can acquire it.</string>
     <string name="region_onlinecity_acquired">This is your plot. You can found a city here.</string>
     <string name="region_onlinecity_wait">Wait some time before you acquire another city in that region.</string>
     <string name="dialog_savingissue_title">Saving failed</string>
     <string name="dialog_savingissue_text">Unfortunately your city could not be saved.\nHint: %s\n\nPlease try again.</string>
+    <string name="dialog_savingissue_cmdcancel">Cancel</string>
+    <string name="dialog_savingissue_cmdretry">Retry</string>
```

Comment:
Are these actual translation strings? If not, consider making a feature request.